doc/HOWTO_feature_flags.md

```diff
@@ -0,0 +1,56 @@
+# Feature Flags
+
+## Style/Display
+
+* Manage facilities link on the left or right
```

Comment:
I realize this is from my notes :) any idea what feature flag this is referencing?

Reply:
After a bit of digging, looks like this is the `use_manage` flag, which seems to turn "Home" to "Use" and moves "Manage Facilities" to the right or left.

Everyone has this set to `false`. This feature seems poorly implemented because "Home" is hardcoded in several places even though the feature flag changes the "Home" navigation item to "Use"

https://github.com/tablexi/nucore-open/blob/8bef19f15de7ec0a2db032fc21aabfcead77f91f/config/settings.yml#L117

https://github.com/tablexi/nucore-open/blob/8bef19f15de7ec0a2db032fc21aabfcead77f91f/app/views/shared/nav/_nav_links.html.haml#L4

Reply:
Nice detective work! 🔎